app/routes/task_routes.py

```diff
@@ -1 +1,151 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response
+from ..db import db
+from ..models.task import Task
+from .route_utilities import validate_model
+import json
+from datetime import datetime, timezone
+import requests
+import os
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+
+@tasks_bp.post("")
+def create_task():
+
+    request_body = request.get_json()
+
+    try:
+        new_task = Task.from_dict(request_body)
+    except KeyError as e:
+        if e.args[0] == "title" or e.args[0] == "description":
+            response = {"details": "Invalid data"}
```

Comment:
Same comment as above in `create_goal` about adding more details to `response` so the client knows what they should fix up in their incorrect request body